benchmarks/nemo/tracer_advection/original/Makefile

```diff
@@ -2,9 +2,12 @@
 # targets for building the original (serial) version and three
 # different PSyclone-processed versions:
 #
-#  1. OpenACC;
-#  2. OpenMP for CPU;
-#  3. OpenMP offload for GPU.
+#  1. Serial;
+#  2. OpenACC kernels for GPU with UniMem;
+#  3. OpenACC loops for GPU with UniMem;
+#  4. OpenMP k-loops parallelism for CPU;
+#  5. OpenMP all-loops parallelism for CPU;
+#  6. OpenMP 5 all-loops parallelism for GPU with UniMem.
```

Comment:
When you say "all-loops" here do you mean all outer loops?

Reply:
changed to "outer-"

Reply:
Yes, that's clear now I think.